src/main/resources/git-commit-id.properties

```diff
@@ -0,0 +1 @@
+git-commit-id-plugin.version=${version}
```

Comment:
Is this file useful?

Reply:
This file is useless, so I delete it from the repository.